kotest-assertions/kotest-assertions-json/src/commonTest/kotlin/io.kotest.assertions.json/schema/ArraySchemaTest.kt

```diff
@@ -100,5 +100,54 @@ class ArraySchemaTest : FunSpec(
             $ => Sequence should be Unique
          """.trimIndent()
       }
+
```

Comment:
I would like to add a test with a more complex structure. Preferably with an array with objects with a contains-spec that fails all objects in the array, so we can see that the error message provided is useful